src/components/molecules/definitionitem/DefinitionItem.jsx

```diff
@@ -4,15 +4,41 @@ import React from 'react';
 import './DefinitionItem.scss';
 
 import { SynonymTable } from '../../atomic';
+import { useStateValue } from '../../../store/store.js';
 
-const DefinitionItem = ({ payload }) => {
-	const drugInfoSummaryLink = payload?.drugInfoSummaryLink;
-	const definitionText = payload.definition?.html;
+const DefinitionItem = ({
+	drugInfoSummaryLink,
+	definitionText,
+	nciConceptId,
+	termId,
+	aliases,
+	name,
+}) => {
+	const [
+		{ nciThesaurusConceptLink, ctsDynamicListingPageBase },
+	] = useStateValue();
 
+	const renderSupplementalLinks = () => {
+		return (
+			<div role="contentinfo">
```

Comment:
this isn't a proper role for this.  please remove role and add `class="supplemental-links"`

Reply:
actually, maybe this should be `class="term-definition__supplemental-links"` following BEM.  but then you would also have to look at renaming other things like, `term-definition__synonym-table`, `term-definition__term`, `term-definition__definition`, etc...

Reply:
I'd assume this goes for search-box and other items/views..